### DISCUSSION TITLE
New API, Next.js and a CMS

### DISCUSSION THREAD

**Discussion by dan-lindley** [Budibase/budibase#4983] [Ideas]
So i'm new and i'm not a developer. Be Kind!

I've seen the release with RESTful API etc and that a budibase app can be used as a backend and it can be linked to Next.js. So I looked at this and it seems that also links into the usual CMS platforms. so is a budibase > Next > WP/Drupal etc link all possible?

I can follow budibase and create the stuff i need to make an idea of mine work. The templates are a breeeze to work through and i lovwe it to be honest. However ive looked at the various CMS plugins and I cant find one that quite handles data like budibase does. It's like youre having to set tables up as "content types" that never really fit the bill.

I'd love to have the app designed in budibase and the data be pulled into a CMS which could manage other things like users and user groups. Any ideas?

**Comment by mjashanks** [comment]
Hey @dan-lindley.

The Budibase API enables Budibase to be used as a Backend. It sounds as though you are wanting to store your data elsewhere (in CMS), and use Budibase to connect into that. This is the reverse - i.e. Budibase as a frontend.

You should already be able to do this by adding a "REST" data connector, and connecting to your CMS that way.

**Comment by dan-lindley** [comment]
Hi @mjashanks 

No sorry, i dont think i was clear. I want to build my data storage in budibase and have that keep hold of it because i like budibase and it makes sense. 
I'd like to be able to have the CMS just manage users/user groups only. The CMS would display either - whole screens from budibase or have the data plugin into pages maybe by shortcode or something?

I've been struggling to create something simple. I can't deploy Budibase on my shared hosting :( and i really am uncertain how to proceed. All i wanted to do was have a database to store records. Records would be related to each other. Users would be in groups and manage only their own data but able to view data within their own group. 

to me this sounds very easy and something i'd have been able to achieve in the 90s/00s on access!!!!! :/

Dan
